ci-operator/jobs/openshift/hypershift/openshift-hypershift-release-4.14-periodics.yaml

```diff
@@ -880,14 +880,7 @@ periodics:
     ci.openshift.io/generator: prowgen
     job-release: "4.14"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-kubevirt-metal-mce
-  reporter_config:
-    slack:
-      channel: '#hypershift-kubevirt-platform'
-      job_states_to_report:
-      - failure
-      report_template: ':volcano: Job *{{.Spec.Job}}* ended with *{{.Status.State}}*.
-        <{{.Status.URL}}|View logs> :volcano:'
```

Comment:
For whatever reason, when we re-name a job, the slack reporting gets stripped out of the job yaml. This needs to be added back